client/my-sites/customer-home/index.js

```diff
@@ -4,6 +4,11 @@ import { navigation, siteSelection, sites } from 'calypso/my-sites/controller';
 import renderHome, { maybeRedirect } from './controller';
 
 export default function () {
+	// Redirect old view routes to home
```

Comment:
Do you think we might be able to remove these altogether after updating the remaining links like in #102930 ?

Reply:
In theory we could if we have no more links left, but the `/view` routes seemed like primary routes in the past so I wonder if folks or search engines or random links could still exist out there.